front-end/src/components/Pages/Billing.js

```diff
@@ -40,33 +42,39 @@ class Billing extends Component {
                     type="text"
                   />
                 </div>
+                </FormItem>
               </div>
               <div className="card_body">
-                <div>
-                  <h3>Expires on</h3>
-                  <input
-                    value={this.state.EXP}
-                    onChange={this.inputChangeHandler}
-                    name="EXP"
-                    type="text"
-                  />
-                </div>
+                <FormItem>
+                    <div>
+                      <h3>Expires on</h3>
+                      <input
+                        value={this.state.EXP}
+                        onChange={this.inputChangeHandler}
+                        name="EXP"
+                        type="text"
+                      />
+                    </div>
+                  </FormItem>
               </div>
               <div className="card_body">
-                <div>
-                  <h3>CCV</h3>
-                  <input
-                    value={this.state.CCV}
-                    onChange={this.inputChangeHandler}
-                    name="CCV"
-                    type="text"
-                  />
-                </div>
+                <FormItem>
+                    <div>
+                      <h3>CCV</h3>
+                      <input
+                        value={this.state.CCV}
+                        onChange={this.inputChangeHandler}
+                        name="CCV"
+                        type="text"
+                      />
+                    </div>
+                  </FormItem>
               </div>
               <button className="billing_btn">Buy Now</button>
```

Comment:
Will be turned into styled component. You can make this if you want to build a button.